src/test/java/com/google/api/codegen/grpcmetadatagen/testdata/library_pkg.yaml

```diff
@@ -28,18 +30,24 @@ gax_version:
     upper: '0.16.0'
   nodejs:
     lower: '^0.7.0'
+  java:
+    lower: '0.6.1'
```

Comment:
Gax version is a required field of the metadata view. I can mark the field nullable but I feel we can just leave the version here since it would be used in the near future anyway.